README.md

```diff
@@ -152,7 +152,8 @@ string_params_with_assocs(:comment, attrs)
 # Example of use in Phoenix with a factory that uses ExMachina.Ecto
 defmodule MyApp.MyModuleTest do
   use MyApp.ConnCase
-  # You can also import this in your MyApp.ConnCase if using Phoenix
+  # You can also import this in your MyApp.ConnCase (inside the using block)
+  # if using Phoenix
```

Comment:
Maybe we could shorten this to `If using Phoenix, import this inside the using block in MyApp.ConnCase`. I think that should fit on one line